docs/building-spark.md

```diff
@@ -272,3 +272,31 @@ For SBT, specify a complete scala version using (e.g. 2.12.6):
     ./build/sbt -Dscala.version=2.12.6
 
 Otherwise, the sbt-pom-reader plugin will use the `scala.version` specified in the spark-parent pom.
+
+## Running Jenkins tests with Github Enterprise
+
+To run tests with Jenkins:
+
+    ./dev/run-tests-jenkins
+
+If use an individual repository or an GitHub Enterprise, export below environment variables before running above command.
```

Comment:
nit: `an` -> `a`
In addition, how about `an GitHub Enterprise` -> `a repository on GitHub Enterprise`?